users/helauth/parsers.py

```diff
@@ -40,11 +50,46 @@ def parse(self) -> ReservationPrefillInfo:
             home_city=self.get_user_home_city(),
         )
 
-    def get_last_name(self) -> str | None:
-        return self.data.get("lastName")
+    def parse_birthday_info(self) -> BirthdayInfo:
+        return BirthdayInfo(
+            id=self.data.get("id"),
+            birthday=self.get_birthday(),
+        )
+
+    def parse_user_profile_info(self, *, user: User) -> UserProfileInfo:
```

Comment:
In the case of `parse_reservation_prefill_data` it is indeed desirable to prefer the profile level (user editable) name & address data as now, but in the added `parse_user_profile_info`, if `verifiedPersonalInformation` is present, then firstName/lastName and permanent(Foreign)Address should be attempted first, so that handlers will see the official DVV data of a person if it is available.

Reply:
Done